src/Dialog/__snapshots__/Dialog.test.tsx.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DialogCentreScreen content renders English copy 1`] = `<div />`;
+
+exports[`DialogCentreScreen content renders French copy 1`] = `<div />`;
```

Comment:
This is the snapshot, just a `<div />` haha. Let's just remove the snapshot tests altogether, it just adds more time to tests without adding any value.